test/unittest/security/authentication/BuiltinPKIDHTests.cpp

```diff
@@ -402,6 +402,75 @@ void AuthenticationPluginTest::check_shared_secrets(
     ASSERT_TRUE(*sharedsecret_1 == *sharedsecret_2);
 }
 
+TEST_F(AuthenticationPluginTest, validate_local_identity_kagree_algo)
+{
+    const std::string correct_values[] =
+    {
+        "RSA",
```

Comment:
This would apply if the previous suggestion in correct
```suggestion
        "DH",
```

Reply:
1babae4